app/controllers/admin/facility_groups_controller.rb

```diff
@@ -60,7 +58,14 @@ def set_protocols
 
   def set_facility_group
     @facility_group = FacilityGroup.friendly.find(params[:id])
-    authorize([:manage, @facility_group])
+  end
+
+  def authorize_facility_group
+    raise Pundit::NotAuthorizedError unless current_admin.can_manage_facility_groups?(@facility_group)
+  end
+
+  def authorize_organization
+    raise Pundit::NotAuthorizedError unless current_admin.can_manage_organizations?(@facility_group.organization)
```

Comment:
Stylistic thing - using the suffix version of unless is not ideal here, because it kinda buries the critical piece of this logic.  Some alternatives:

```ruby
def authorize_facility_group
  unless current_admin.can_manage_organizations?(@facility_group.organization)
    raise NotAuthorized
  end
end
```

Shorter, keeps the predicate check front and center

```ruby
def authorize_facility_group
  current_admin.can_manage_organizations?(@facility_group.organization) || raise NotAuthorized
end
```

Or if we want the model to tell us about it, we could add an API like this. If we are doing the raise style everywhere, I think this would be nice to add.  Note that we don't need to tell the user what kind of object we have, because the relationship is polymorphic.  The Access model should be able verify access for any type of allowable region.

```ruby
def authorize_facility_group
  current_admin.verify_manage_access!(organization)
end

class User
  def verify_manage_access!(region)
    accesses.admin.find_by(resource: region) || raise NotAuthorized
  end
end
```